commit 50fdd8bd6ab8715743283d6cd4045875bbef9bc1
Author: Joshua Goins <josh@redstrate.com>
Date:   Wed Nov 27 19:03:16 2024 -0500

    Always use a ToolBar for pages on the stack
    
    On mobile, usage of the stack is quite common. For example, opening the
    settings in Tokodon. However, due to this check - page actions disappear
    because it's in Title mode.
    
    Instead, we should always use the ToolBar style when determining the
    global ToolBarStyle for a stack. This makes page actions show up on
    mobile when pushing pages via the layers system.

diff --git a/src/controls/Page.qml b/src/controls/Page.qml
index 06bf440c..29dcf2bb 100644
--- a/src/controls/Page.qml
+++ b/src/controls/Page.qml
@@ -138,7 +138,7 @@ QQC2.Page {
         if (globalToolBar.row && !globalToolBar.stack) {
             return globalToolBar.row.globalToolBar.actualStyle;
         } else if (globalToolBar.stack) {
-            return Kirigami.Settings.isMobile ? Kirigami.ApplicationHeaderStyle.Titles : Kirigami.ApplicationHeaderStyle.ToolBar;
+            return Kirigami.ApplicationHeaderStyle.ToolBar;
         } else {
             return Kirigami.ApplicationHeaderStyle.None;
         }
